lld/COFF/Config.h

```diff
@@ -313,6 +313,7 @@ struct Configuration {
   bool warnDebugInfoUnusable = true;
   bool warnLongSectionNames = true;
   bool warnStdcallFixup = true;
+  bool warnExportedDllMain = true;
```

Comment:
This is an ABI break.

Reply:
I don't think this is an installed header though?

Reply:
Removed the field in this backport.